lib/handlers/connect.js

```diff
@@ -60,28 +60,43 @@ function authenticate (arg, done) {
     negate)
 
   function negate (err, successful) {
+    var errorMessages = [
+      '',
+      'unacceptable protocol version',
+      'identifier rejected',
+      'Server unavailable',
+      'bad user name or password',
+      'not authorized'
+    ]
```

Comment:
Can you move this to the top o the file?

Reply:
done 👍